edge_computing/day_2_core_cnf_clusters/security/telco-security-sec-context-constraints.adoc

```diff
@@ -0,0 +1,96 @@
+:_mod-docs-content-type: ASSEMBLY
```

Comment:
I think you can consider creating a module for some of the info present in this assembly.